app/helpers/feeds_helper.rb

```diff
@@ -0,0 +1,9 @@
+module FeedsHelper
+  def tweet_oembed(tweet)
+    @twit_init.client.oembed(tweet.tw_id_str, { omit_script: true })
+  end
+
+  def gram_oembed(post)
+    HTTParty.get("http://api.instagram.com/oembed?url=http://instagr.am/#{post.link[22,20]}&maxwidth=500")
```

Comment:
The numbers in your `post.link` reference are a little magical. It'd be worth defining descriptive vars so future developers will know the how and why of the values